dev/devicelab/manifest.yaml

```diff
@@ -279,6 +279,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  complex_layout_scroll_perf__devtools_memory:
+    description: >
+      Measures memory usage of the scroll performance test using DevTools.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```

Comment:
oh, one other nit: may be worth starting this out as `flaky` so if it fails initially you can fix it without needing to revert.

Reply:
(or otherwise block the tree)